src/docs/build-deploy-and-maintain-apps/build-an-application.md

```diff
@@ -40,115 +41,88 @@ This document outlines the best practices for building applications on OpenShift
 ## Requirements to build your application 
 Here are ten common practices for building applications in a cloud native way that we strongly suggest your team follow before building an application on OpenShift:
 
-1. **Learn OpenShift basics**: Acquaint your team with OpenShift's core concepts including projects, pods, services, routes and deployments. Find out more information about [OpenShift training](https://digital.gov.bc.ca/cloud/services/private/support/#platform) 
+1. **Learn OpenShift basics**: Understand OpenShift's core concepts including projects, pods, services, routes and deployments. Sign up for the [OpenShift training](https://digital.gov.bc.ca/cloud/services/private/support/#platform)
 
-2. **Understand application requirements**: It is important that before you build  your application your team clearly defines its requirements, including dependencies, external services and resource needs.  This is particularly important in order to plan for scalability and high availability of your application
+2. **Define application requirements**: Clearly define its requirements, including dependencies, external services and resource needs for scalability and high availability
```

Comment:
```suggestion
2. **Learn Kubernetes fundamentals**: OpenShift is built on Kubernetes, so basic knowledge is essential. Get comfortable using the [Kubernetes documentation](https://kubernetes.io/docs/home/).
```

Reply:
I think it makes sense to swap these two because it would be easier to define app requirements after the team already knows a little about the environment they'll be working in.